deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -12,7 +12,28 @@ products:
 
 A [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
 
-ECK manages a default PDB per {{es}} resource. It allows one {{es}} Pod to be taken down, as long as the cluster has a `green` health. Single-node clusters are not considered highly available and can always be disrupted.
+ECK manages either a single default PDB, or multiple PDBs per {{es}} resource according to the license available.
```

Comment:
Does the user have to be on a stack 9.2 cluster as well as use ECK 3.2? 

Consider this approach. More tagging / clarification might be necessary if the stack version is also a factor.

We have a couple of other ways we could do this with heading tagging or tabs, but this is probably a nice lightweight way to do it

```suggestion
ECK manages either a single default PDB, or multiple PDBs per {{es}} resource according to the license available.

:::{note}
In ECK 3.1 and earlier, all clusters follow the [non-enterprise behavior](#non-enterprise-licensed-customers), regardless of license type.
:::
```